app/src/main/java/com/github/fribourgsdp/radio/MainActivity.kt

```diff
@@ -26,6 +26,5 @@ class MainActivity : AppCompatActivity() {
             startActivity(Intent(this, UserProfileActivity::class.java))
         }
 
-        User("bla", User.generateColor()).save(this)
```

Comment:
this is still necessary for master as it will be used in the demo. You can remove it and I will add it back in my PR